### PR TITLE
[improvement](bitmap) support version for ser/deser of bitmap

### PR DESCRIPTION
**Pull request by jacktengg**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

File: be/src/util/bitmap_value.h [hunk 1]
```diff
@@ -1159,6 +1173,10 @@ inline Roaring64MapSetBitForwardIterator Roaring64Map::end() const {
 class BitmapValueIterator;
 class BitmapValue {
 public:
+    enum BitmapSerializeVersion {
```
Comment (yiguolei) [inline be/src/util/bitmap_value.h:1176]: Is this useful?

Comment (jacktengg) [reply be/src/util/bitmap_value.h]: fixed

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by jacktengg** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**